index.js

```diff
@@ -1,57 +1,75 @@
-const addBookBtn = document.querySelector('.add-book-btn');
+const listElm = document.querySelector('.books-list');
 const form = document.querySelector('.form');
-const listItem = document.querySelector('.books-list');
 
-let booksList = localStorage.getItem('awbooks')
-  ? JSON.parse(localStorage.getItem('awbooks'))
-  : [];
+class Books {
+  constructor() {
+    this.list = localStorage.getItem('booksList')
+      ? JSON.parse(localStorage.getItem('booksList'))
+      : [];
+  }
 
-function addBook(book) {
-  booksList.push(book);
+  addBook(book) {
+    this.list.push(book);
 
-  localStorage.setItem('awbooks', JSON.stringify(booksList));
-}
-function removeBook(book) {
-  booksList = booksList.filter((currBook) => currBook.title !== book.title);
-  localStorage.setItem('awbooks', JSON.stringify(booksList));
+    localStorage.setItem('booksList', JSON.stringify(this.list));
+  }
+
+  removeBook(title) {
+    // if (book) {
+    this.list = this.list.filter((currentBook) => currentBook.title !== title);
+
+    localStorage.setItem('booksList', JSON.stringify(this.list));
+    // }
```

Comment:
- [ ]  Please kindly remove this old piece of commented code so the repo gets cleaner 💪